javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/scripts.js

```diff
@@ -0,0 +1,231 @@
+
+
+function app() {
+
+//  Songs array creation
+
+const songs = [
+    { title: "Scratching The Surface", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",artist: "Quincy Larson", duration: "4:25", src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/scratching-the-surface.mp3"},
+    { title: "Can't Stay Down", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "4:15" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stay-down.mp3"},
+    { title: "Still Learning", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:51" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/still-learning.mp3"},
+    { title: "Cruising for a Musing", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:34" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cruising-for-a-musing.mp3"},
+    { title: "Never Not Favored", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:35" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/never-not-favored.mp3"},
+    { title: "From the Ground Up", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:12" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/from-the-ground-up.mp3"},
+    { title: "Walking on Air", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:25" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/walking-on-air.mp3"},
+    { title: "Can't Stop Me. Can't Even Slow Me Down.", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:52" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stop-me-cant-even-slow-me-down.mp3"},
+    { title: "The Surest Way Out is Through", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:10" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/the-surest-way-out-is-through.mp3"},
+    { title: "Chasing That Feeling", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "2:43" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/chasing-that-feeling.mp3"},
+  ];
+
+  // song playlist
+const songListDiv = document.getElementById("playlist__songs");
+const closeBtn = document.querySelector(".playlist__close");
```

Comment:
We haven't really been following the BEM methodology in either of the new JavaScript or new Responsive web design curriculum. 
We have mainly been using kebab case. 

Personally I like to use BEM when working with SASS but if this hasn't been taught in the Responsive web design curriculum then we should stick with kebab case here and change it in the html and css files too

```suggestion
const songListDiv = document.getElementById("playlist-songs");
const closeBtn = document.querySelector(".playlist-close");
```